### ISSUE TITLE
Compiler warnings : start value has mixed support, consider using flex-start instead`

### ISSUE THREAD

**Issue by sven09** [Upmostly/modali#8]
Hi,
love this component.

Got a compiler error, while using it with Typescript.

```
Compiled with warnings.

./node_modules/modali/dist/modali.css (./node_modules/css-loader/dist/cjs.js??ref--6-oneOf-3-1!./node_modules/postcss-loader/src??postcss!./node_modules/modali/dist/modali.css)
Warning

(39:3) start value has mixed support, consider using flex-start instead
```

**Comment by jcrowson** [comment]
@sven09 Thanks! Accepted and I'll push a fix ASAP.

**Comment by coffenbacher** [comment]
I'm also seeing this error, has it been fixed? I'm on version 1.1.0

**Comment by jcrowson** [comment]
@coffenbacher @sven09 I'm pushing a fix in v1.1.1 which will be published in the next few hours 👍
